libbindgen/tests/headers/inline_namespace.hpp

```diff
@@ -0,0 +1,11 @@
+// bindgen-flags: --enable-cxx-namespaces -- -std=c++11
+
+namespace foo {
+  inline namespace bar {
+    using Ty = int;
+  };
```

Comment:
What are the C++ semantics for when there are conflicts between the inline namespace and its parent namespace? If it is permitted at all, and not a compilation error, then we should test that we Do The Right Thing in this scenario.

```c++
namespace foo {
  inline namespace bar {
    using Ty = int;
  }
  using Ty = char;
}
```

http://stackoverflow.com/questions/24220127/redefinition-member-of-the-namespace-into-the-nested-inline-namespace#24220504 seems relevant